python-modules/README.md

```diff
@@ -0,0 +1,120 @@
+# Mozilla Change Integration Python Modules
+
+Mozilla IAM uses a variety of python modules to achieve integration
+across the stack.  In version 1 of the change integration service one library
+formerly known as `cis` held all this functionality.  In v2 of the service separation of concerns requires a more loosely coupled approach.
+
+
+## Modules in this directory.  
+
+**cis-common**
+
+Code that is common to all additional libs.  Configuration management, authorization layers, etc.
+
+_User Stories_
+
+As a publisher I must exchange a CLIENT_ID and CLIENT_SECRET with auth0 for a bearer token.  This grants the ability to publish changes.
```

Comment:
`access token`
i know the text in the header says bearer, but it's an access token =)